recipes/splinecalib/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "splinecalib" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cfd734701977949dde328712f9e4c578eefb21eb8295a6c3fb7462a7b2ee5083
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.16

```

Comment:
Remove the min version to let conda use the version specified in [pinning-feedstock](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml)